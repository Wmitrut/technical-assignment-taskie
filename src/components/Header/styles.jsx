import styled from 'styled-components';
import { COLORS } from '../../constants/colors';

export const Container = styled.header`
  width: 100%;
  height: 56px;
  background: ${COLORS.primaryDark};
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.1);
  padding: 10px;
  box-sizing: border-box;
  display: flex;
  align-items: center;
  justify-content: flex-start;
  font-family: Inter;
  font-weight: bold;
`;
