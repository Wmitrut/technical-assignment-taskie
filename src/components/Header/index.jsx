import { Container } from './styles';

const Header = () => {
  return (
    <Container>
      <p>Taskie</p>
    </Container>
  );
};

export default Header;
