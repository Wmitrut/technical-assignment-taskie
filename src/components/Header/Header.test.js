import renderer from 'react-test-renderer';

import Header from './index';

it('Renders header correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
