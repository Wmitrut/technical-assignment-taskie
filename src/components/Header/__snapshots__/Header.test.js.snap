// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders header correctly 1`] = `
<header
  className="sc-bczRLJ eqqKqN"
>
  <p>
    Taskie
  </p>
</header>
`;
